These git commands now error out, instead of returning aports info.

--- a/Makefile
+++ b/Makefile
@@ -46,8 +46,8 @@
 	$(CC) $(ALL_CFLAGS) -c $< -o $@
 
 gitversion.h:
-	echo "#define GIT_COMMIT_ID 0x`git rev-parse HEAD | head -c8`" > $(PWD)/gitversion.h
-	echo "#define GIT_COMMIT_COUNT `git rev-list --count --all`" >> $(PWD)/gitversion.h
+	echo "#define GIT_COMMIT_ID 0x0" > $(PWD)/gitversion.h
+	echo "#define GIT_COMMIT_COUNT 0" >> $(PWD)/gitversion.h
 
 # Static mbed TLS
 mbedtls: $(MBEDTLS_CONFIG)
